.changeset/shy-bobcats-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/fields': patch
+---
+
+Fix virtual field rendering of false & 0 values
```

Comment:
```suggestion
Fixed virtual field rendering of false & 0 values.
```

Reply:
We should write this down somewhere obvious, but our convention is to write our changeset descriptions in past tense, so that they make sense when a person in the future is reading the CHANGELOG file.